### PR TITLE
modified exportinstance to contain new integration type property

### PR DESCRIPTION
**Pull request by RynhardPietersen**
## Technical Summary

Link: https://dimagi-dev.atlassian.net/browse/SS-233
Modified ExportInstance to allow for use with the google sheet integration function just like how odata uses it.

### Safety story

Local testing. Only changed up an object and some JS.

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

**Review by biyeun — APPROVED** [review]
This looks good so far. I'd want to see this branch up on staging first :)

Then of course we need to coordinate a good time to deploy and reindex. cc @dannyroberts if you have any thoughts/input on this?

**Review by dannyroberts — COMMENTED** [review]
My first thought (and maybe this is naive) is, would it be simpler to treat this as a new "export_format" rather than a different field, while still meeting the need? My reasoning is that (I think) you'll always _either_ use google sheet integration _or_ have to pick an export format like xlsx, xls, csv, etc., and so functionally "google sheets" _is_ itself a kind of export format. Doing this would also make it so a reindex is not required.

**Comment by biyeun** [comment]
Oh, that is a fantastic idea @dannyroberts! @RynhardPietersen Let's quickly sync Friday to digest this suggestion together.

File: corehq/apps/export/models/new.py [hunk 1]
```diff
@@ -735,7 +735,6 @@ class CaseExportInstanceFilters(ExportInstanceFilters):
 class FormExportInstanceFilters(ExportInstanceFilters):
     user_types = ListProperty(IntegerProperty, default=[HQUserType.ACTIVE, HQUserType.DEACTIVATED])
 
-
```
Comment (biyeun) [inline corehq/apps/export/models/new.py:738]: might be good to uncommit this white space

**Review by biyeun — APPROVED** [review]
looks good. just a couple nits on whitespace removals being committed unintentionally

**Review by biyeun — APPROVED** [review]

**Pull request merged**